cfy_manager/utils/service.py

```diff
@@ -147,12 +174,20 @@ def _get_full_service_name(service_name, append_prefix):
 
 
 class Supervisord(object):
-    def supervisorctl(self, action, service='', ignore_failure=False):
+    def supervisorctl(
+            self,
+            action,
+            service='',
+            ignore_failure=False,
+            options=[]
```

Comment:
OK, I missed this further up, but this needs to be non-mutable. Probably just set it as None?

Reply:
(and the same on the other list-as-default-arg)

Reply:
Done